ConsoleGame.java

```diff
@@ -0,0 +1,64 @@
+import java.io.*;
+
+public class ConsoleGame implements GameType {
+
+    public ConsoleGame() {
+    }
+
+    @Override
+    public String drawBoard(Board board) {
+        print("|" + board.getState().substring(0, 3) + "|\n|" + board.getState().substring(3, 6) + "|\n|" + board.getState().substring(6, 9) + "|\n");
+        return String.format("|" + board.getState().substring(0, 3) + "|\n|" + board.getState().substring(3, 6) + "|\n|" + board.getState().substring(6, 9) + "|\n");
+    }
+
+    @Override
+    public void drawTurn(Game game) {
+        print("----------------- \n");
+        print("Its your turn " + game.playerActive().getMark() + "!\n");
+        print("----------------- \n");
+    }
+
+    public void print(String text) {
+        byte[] stringByte = text.getBytes();
+        ByteArrayOutputStream bos = new ByteArrayOutputStream(text.length());
```

Comment:
This is composition, but it's a dependency and should be injected into the ConsoleGame constructor.  Makes for easier testing if you wanted to create a mock for IO.